File: repos_estudiantes_ALVAREZ_primera_entrega/APAZA_SEGA_LEONEL_ISAIAS/Herencia/1.py

class Persona:
    def __init__(self, nombre, ci):
        self.nombre = nombre
        self.ci = ci

class Estudiante(Persona):
    def __init__(self, nombre, ci, carrera):
        super().__init__(nombre, ci)
        self.carrera = carrera

    def mostrar(self):
        print(f"Estudiante: {self.nombre} | CI: {self.ci} | Carrera: {self.carrera}")

# Ejemplo
e1 = Estudiante("Ana López", 123456, "Informática")
e1.mostrar()

File: repos_estudiantes_ALVAREZ_primera_entrega/APAZA_SEGA_LEONEL_ISAIAS/Herencia/1.java
class Persona {
    protected String nombre;
    protected int ci;

    public Persona(String nombre, int ci) {
        this.nombre = nombre;
        this.ci = ci;
    }
}

class Estudiante extends Persona {
    private String carrera;

    public Estudiante(String nombre, int ci, String carrera) {
        super(nombre, ci);
        this.carrera = carrera;
    }

    public void mostrar() {
        System.out.println("Estudiante: " + nombre + " | CI: " + ci + " | Carrera: " + carrera);
    }

    public static void main(String[] args) {
        Estudiante e1 = new Estudiante("Ana López", 123456, "Informática");
        e1.mostrar();
    }
}
